src/utils/tokens.ts

```diff
@@ -356,14 +356,11 @@ const tokens = {
     // We might have transactions where the nano contract will pay for deposit fees
     // so we must consider the skipDepositFee flag to skip the utxo selection
     if (!skipDepositFee) {
```

Comment:
```suggestion
    if (!skipDepositFee) {
      depositAmount += this.getTransactionHTRDeposit(amount, data?.length ?? 0, storage);
```

And we can remove the rest of the if, right?